### ISSUE TITLE
TextBox: Always show the label and deactivate the animation

### ISSUE THREAD

**Issue by Stensan** [AvaloniaUI/Avalonia#13245]
## Describe the solution you'd like
Please add the functionality for the TextBox to always show the label and to deactivate the animation. For TextInputLayout in Android we have these two attributes for this: app:expandedHintEnabled="false" and app:hintAnimationEnabled="false".

**Comment by maxkatz6** [comment]
I am not sure if it is worth of adding to the framework, as it should be easy to implement for specific application via styles.
I.e. as easy as single style:
```xaml
    <Style Selector="TextBox.myTextBox /template/ TextBlock#PART_FloatingWatermark">
      <Setter Property="IsVisible" Value="True" />
    </Style>
```
Any text box with `Classes="myTextBox"` will have a floating watermark (label) always visible.

For more about Avalonia styles:
https://docs.avaloniaui.net/docs/next/basics/user-interface/styling/styles

**Comment by Stensan** [comment]
> I am not sure if it is worth of adding to the framework, as it should be easy to implement for specific application via styles. I.e. as easy as single style:
> 
> ```
>     <Style Selector="TextBox.myTextBox /template/ TextBlock#PART_FloatingWatermark">
>       <Setter Property="IsVisible" Value="True" />
>     </Style>
> ```
> 
> Any text box with `Classes="myTextBox"` will have a floating watermark (label) always visible.
> 
> For more about Avalonia styles: https://docs.avaloniaui.net/docs/next/basics/user-interface/styling/styles

Thanks for the answer. I inserted the style in my App.axaml and use Classes="myTextBox" for my TextBox, but it doesn't seem to work. I use MaterialTheme, if this is important. I'm pretty new to Avalonia.
<TextBox Name="Quantity" assist:TextFieldAssist.Label="Quantity" UseFloatingWatermark="True" Classes="myTextBox" Theme="{StaticResource FilledTextBox}" Margin="5"/>
![Screenshot 2023-10-18 111509](https://github.com/AvaloniaUI/Avalonia/assets/36670367/3b6d361e-a8de-4887-9eb1-fecd62b3cd39)

**Comment by timunie** [comment]
check Material styles to find out the right selector they use. 

In that case, I guess it would be a valid solution for you and we can convert this issue into a discussion?

**Comment by Stensan** [comment]
> check Material styles to find out the right selector they use.
> 
> In that case, I guess it would be a valid solution for you and we can convert this issue into a discussion?

Yes, you can do that. I opened a discussion at Material.Avalonia.
